README.md

```diff
@@ -56,9 +56,15 @@ The `<template_location>` parameter is one of these two things:
   one of the inputs declared by the template in its `spec.yaml`. May be repeated
   to provide multiple inputs, like
   `--input=name=alice --input=email=alice@example.com`.
-- `--input-file=file`: provide a yaml file with input(s) to the template. Each `key` 
-  must have a string `val` (i.e. no nesting and no arrays). If `key` exists in the 
-  file but is also provided as an input parameter, the input value takes precedence.
+- `--input-file=file`: provide a YAML file with input(s) to the template. The
+  file must contain a YAML object whose keys and values are strings. If a key
+  exists in the file but is also provided as an `--input`, the `--input` value
+  takes precedence.
+
+  This flag may be repeated, like
+  `--input-file=some-inputs.yaml --input-file=more-inputs.yaml`. When there are
+  multiple input files, they must not have any overlapping keys.
```

Comment:
What's the reason for not supporting overlapping keys vs. allowing later files to override keys? Seems easier to implement the latter (unless there's a design principle I'm missing here).

e.g. `--input-file=defaults.yaml --input-file=overrides.yaml`

Reply:
If this happens, I think it's likely to be a mistake and I want to help the user not do something they don't intend. I don't think the "defaults plus overrides" pattern will occur much in real use cases (do you have any concrete use cases in mind?)

Reply:
No strong opinion. Like I mentioned, it's a design decision that I wanted to understand.